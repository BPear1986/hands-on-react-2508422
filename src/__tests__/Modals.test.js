import React from "react"
import renderer from "react-test-renderer"
import Modals from"../components/Modals"
import cast from"../cast.json"
import { render } from "react-dom"

it("test runs successfuly", ()=>{
  const truth = true
  expect(truth).toEqual(true)
})

cast.forEach(({memberInfo}) => 
  it("renders a modal with the members info",() => {
  const modal = renderer.create(<Modals member={memberInfo}/>)
  expect(modal).toMatchSnapshot()
}))