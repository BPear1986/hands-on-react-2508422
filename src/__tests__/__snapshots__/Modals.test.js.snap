// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a modal with the members info 1`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Aneirin
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Aneirin"
        src="images/aneirin.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Aneirin is the foremost expert of cold climates in planetoids universe-wide, with a particular inquisitiveness in areas colder than -300 F / - 184.4444444 C (hence the profound personal interest in knitting mittens). Having travelled extensively, Aneirin enjoys collecting, receiving, and sending distinctive postcards from all corners of the macrocosm.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 2`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Celestia Dristi
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Celestia Dristi"
        src="images/celestia-dristi.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        As a mobility engineer, Celestia Dristi helps individuals and companies to improve personal and  professional agility and growth in technology and systems. An expert in translating and implementing  needs into functionality, Celestia Dristi has a unique proclivity for solving problems and helping to   avoid future issues.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 3`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Charmaine
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Charmaine"
        src="images/charmaine.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Charmaine loves fossils from every society and planet, and helping to piece together the stories   they tell of an earlier time. Once completing university studies in paleontology with a minor in  swimming arts, Charmaine immediately transitioned to working on the now-famous Dig of Pleoria.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 4`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Emmyloo, "Em"
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Emmyloo, "Em""
        src="images/emmyloo-em.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        If you look up “lexicography” in the dictionary, you will see a photo of Emmyloo. She claims that  her devotion to lexicography began as early as two years old, when she spoke her first full sentence.   Fluent in 94 languages, Emmyloo helps in analyzing relationships among words and the nuances of their   meanings and relationships to other words.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 5`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Gill
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Gill"
        src="images/gill.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Gill carries on generations of tradition in his family as a hydrologist specializing in  underground reservoirs of aquatic-based planetoids. Of note is his revolutionary studies in the   adaptation and accumulation of life in estuaries and cenotes. He volunteers as an instructor in   advanced swimming and walking to young Amphibians.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 6`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Ichabod
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Ichabod"
        src="images/ichabod.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Operations supervisor Ichabod is unwavering in adherence to the perfection of order and function.  Using inborn impassiveness to distracting influences, Ichabod has led several teams to operational  distinction with the advantage of being able to successfully teach precise hoverboard maneuverability.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 7`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Laverne
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Laverne"
        src="images/laverne.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Laverne holds a degree in theoretical physics from Quantumnus University. Having interned for the  great theoretical physicist, Bob, Laverne pursued a natural aptitude in understanding relationships   among dissimilar measurable quantities to become the theoretical physics coordinator at Quantumnus U.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 8`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Mindigo, 'Mindigogo'
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Mindigo, 'Mindigogo'"
        src="images/mindigo.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Transportation Technologist Mindigo earned the nickname “MindiGoGo” because of her tireless work   in advanced travel and commodity conveyance. After earning a degree in transportation from  Amethystopia’s premiere technical college, Mindigo quickly rolled up through company ranks from cargo   hauler to conveyance savant. She is recipient of the coveted Presidential Medal of Moving Stuff.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 9`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Nalda
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Nalda"
        src="images/nalda.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Autodidact Nalda began work as a mind reader and universal translator in 2288 when founding the  non-profit, Interplanetary Understanders and Lie Detectors, LLC (IULD). Through IULD, Nalda has   successfully helped several formerly feuding planetary governments to pause, recognize their  similarities, and eventually understand what the other was actually saying, assisting in achieve  lasting peace and collaboration.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 10`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Narf the Fourth
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Narf the Fourth"
        src="images/narf-the-fourth.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Narf the Fourth (known to friends as ‘Norf’) oversees the manufacturing of astronomic spaceship  accessories at Saucer & Rocket, Inc. Norf keeps an eye on all aspects of production quality, including  managing several lower quality assurance specialists. Interests include jet-pack racing, bodybuilding,  and volunteer baseball coach. Friends tell us that it’s not summer if you don't hear Norf yelling ”keep   your eye on the ball!"
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 11`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Portman
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Portman"
        src="images/portman.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        No one loves animals, critters, varmints, and beasties as much as Dr. Portman, whether they are  wild or a beloved pet. Leading a crew of superterrestrial veterinarians throughout the galaxy, Dr.  Portman has discovered many antidotes for animal illnesses on a planetoid that can also be used   successfully on other planetoids, and is working to expand the cross-distribution of these treatments   and cures throughout the universe.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 12`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Seymour
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Seymour"
        src="images/seymour.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        “If it isn't counted, it doesn't count!"™ is the mantra Seymour originated and propagated in the   colossal and growing industry of inventory control. Now trademarked as the slogan for Seymour's   company, Totaling, it serves as reassurance of the company's mission and meaning for potential and  current clients. Counting things is pretty much all Seymour does, other than recreational mountain  climbing.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 13`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Tomolor
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Tomolor"
        src="images/tomolor.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        As a professional computer systems analyst, Tomolor has made history in adapting systems for   countless organizations, increasing their overall efficiency by an average of 65%. An active member of  the Beinghood of Bookery, Tomolor has authored several children’s books that explain how to flourish in   their respective areas of interest so that they can most efficiently metamorphose into a career they  love.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 14`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Trisaphron
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Trisaphron"
        src="images/trisaphron.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Trisaphron earned a masters degree in practical operations from Omerania University after  completing a bachelors of science in impractical mathematics. With a keen ability to watch and  understand the nuances of cooperation among production robots, Trisaphron never hesitates to jump in  and lend a hand or two or three to educate and improve procedures.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 15`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Two
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Two"
        src="images/two.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        With growing interest by organizations in expanding their business on multiple planets, Two finds  that it is increasingly critical to focus on sustainability. As Two describes it, we can’t expect any   business to be viable long-term if a planet and its beings aren’t well cared for. Currently stationed   on Earth, Two enjoys collecting and studying lithium batteries.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;

exports[`renders a modal with the members info 16`] = `
<dialog
  id="modal-member"
  open={true}
>
  <article>
    <header>
      <a
        aria-label="Close"
        className="close"
        data-target="modal-member"
        href="#close"
      />
      <span
        style={
          {
            "color": "var(--primary)",
            "fontSize": "2rem",
            "fontWeight": "700",
          }
        }
      >
        Violette
      </span>
    </header>
    <div
      style={
        {
          "display": "flex",
          "gap": "1rem",
        }
      }
    >
      <img
        alt="Violette"
        src="images/violette.svg"
        style={
          {
            "width": "200px",
          }
        }
      />
      <p>
        Violette holds a doctorate degree in Planetary Behaviorism from Fouraluy University. She has been  fascinated in researching the behavior of sentient beings as long as she can remember. She feels her  experience in animal husbandry growing up on her family farm gave her a unique perspective in and   appreciation for all life that drives and complements her current work as a behaviorist. She loves  ballet, classic disco, and spending time with her 17 Earth chickens.
      </p>
    </div>
    <hgroup
      style={
        {
          "display": "flex",
          "gap": "1rem",
          "justifyContent": "space-between",
          "marginTop": "1rem",
        }
      }
    >
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_back
        </span>
      </a>
      <a
        className="outline"
        href="#"
        onClick={[Function]}
        role="button"
      >
        <span
          className="material-symbols-outlined"
        >
          arrow_forward
        </span>
      </a>
    </hgroup>
  </article>
</dialog>
`;
